docs/api-reference/test-utils/browser-test-driver.md

```diff
@@ -4,65 +4,97 @@
 
 # BrowserTestDriver (Test Automation Class)
 
-> Requires Chrome version 64 or higher
+> Note: Requires Chrome version 64 or higher
 
-A higher level helper class (compared to the [`BrowserDriver`](./docs/api-reference/test/browser-task-status) class.) primarily intended for automating browser tests from Node.js shell scripts.
+A higher level helper class that inherits the [`BrowserDriver`](./docs/api-reference/test/browser-task-status). Primarily intended for automating browser tests from Node.js shell scripts.
 
-A `BrowserTestDriver` starts a Chrome browser instance and a server and opens a page with a URL that loads a script from the server. The script that runs in the browser is expected to report status back using the [`callExposedFunction`](./docs/api-reference/test/browser-task-status) class.
+A `BrowserTestDriver` starts a Chromium browser instance and a server and opens a page with a URL that loads a script from the server. The script that runs in the browser is expected to report test results back using predefined global functions.
 
+To use this class, [puppeteer](https://www.npmjs.com/package/puppeteer) must be installed as a dev dependency.
 
 ## Usage
 
 In your node.js start script:
+
 ```js
 // This is the script that runs in Node.js and starts the browser
 const {BrowserTestDriver} = require('probe.gl/test-utils');
 new BrowserTestDriver().run({
-  process: 'webpack-dev-server',
-  parameters: ['--env.browser-test'],
+  server: {
+    command: 'webpack-dev-server',
+    arguments: ['--env.browser-test']
+  },
+  headless: true
 });
 ```
 
 In your script that is run on the browser:
+
 ```js
-// This is the script that runs in Node.js and starts the browser
-import {callExposedFunction} from 'probe.gl/test-utils';
-// Log to terminal console
-callExposedFunction('browserTestLog', 'Test started');
+// Run test cases
 ...
 // App is done running, terminate the browser instance
-callExposedFunction('browserTestComplete', {success: true});
+window.browserTest('done', 'All tests passed');
```

Comment:
Total nit: Not sure the new name is more descriptive/intuitive.

Maybe `window.browserTestDriver.sendMessage(...)`?

Reply:
We can't expose objects, only functions to the global scope. How about `window.sendBrowserTestMessage`?

Reply:
Sure let's go with that style.

Also I had a suggestion (I think in the other diff) regarding exposing multiple functions rather than generic sendMessage with overloaded args and return types. 

It seems to me the more generic `sendMessage` naming approach would be more appropriate if this system is designed to be user extensible on the other side, 

Maybe it is in terms of subclassing these drivers?

Reply:
I agree, once we add another function for diffing it doesn't make sense to have a `sendMessage` func any more. I'm going to split this into two (`browserTestDriver_fail`, `browserTestDriver_finish`).